docs/mkdocs.yml

```diff
@@ -55,6 +55,7 @@ markdown_extensions:
   - pymdownx.details
   - pymdownx.superfences
   - pymdownx.magiclink
+  - pymdownx.inlinehilite
```

Comment:
I had been considering adding this. Have you previewed the docs locally? Did it add much value in syntax highlighting inline code blocks?

Reply:
I am personally using this for my own docs and I can say that I like it. It's mainly useful when you want to distinguish inline Bash and YAML, but you can do more of course, and it does not break anything.

> Have you previewed the docs locally?

I did. It works like a charm.

> Did it add much value in syntax highlighting inline code blocks?

Admittedly, depends on personal taste. I like it.